node/src/types/status_feed.rs

```diff
@@ -12,13 +12,20 @@ pub struct StatusFeed<I> {
     pub last_finalized_block: Option<Block>,
     /// The peer nodes which are connected to this node.
     pub peers: HashMap<I, SocketAddr>,
+    /// The chainspec info for this node.
+    pub chainspec_info: Option<ChainspecInfo>,
```

Comment:
nitpick: given that Option<...> was removed from the chainspec_loader side of this prior to opening the PR, you could also remove it here.

Reply:
Updated it just `ChainspecInfo`